src/renderer/components/Menu.js

```diff
@@ -44,7 +44,7 @@ class Controller {
       var contact = selectedChat.contacts[0]
       confirmation(tx('ask_block_contact'), yes => {
         if (yes) {
-          ipcRenderer.sendSync('dispatchSync', 'blockContact', contact.id)
+          ipcRenderer.send('blockContact', contact.id)
```

Comment:
Apparently, `blockContact` was called both sync and async from two different places. This behavior has now changed since from now on always call it async.